robot_joint_publisher:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

# velocity Controllers ---------------------------------------
LF_WHEEL_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "LF_WHEEL_JOINT"
  pid: {p: 100.0, i: 0.01, d: 10.0}
RF_WHEEL_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "RF_WHEEL_JOINT"
  pid: {p: 100.0, i: 0.01, d: 10.0}

WHEEL_JOINT_ff_param:
  ff_general: 25
  ff_pure_rotation: 100

# The current PID loop
RF_WHEEL_JOINT_dc_controller:
  p: 50.0
  i: 150.0
  d: 0.0
  i_clamp: 300
  antiwindup: True
  publish_state: True

LF_WHEEL_JOINT_dc_controller:
  p: 50.0
  i: 150.0
  d: 0.0
  i_clamp: 300
  antiwindup: True
  publish_state: True
